modules/acm/main.tf

```diff
@@ -14,6 +14,20 @@
  * limitations under the License.
  */
 
+module "enable_acm" {
+  source  = "terraform-google-modules/gcloud/google"
+  version = "~> 2.0"
+
+  platform = "linux"
+  upgrade  = true
```

Comment:
```suggestion
  platform              = "linux"
  upgrade               = true
  additional_components = ["alpha"]
```

Reply:
So sorry for the delay! I totally forgot about this until I came back to the project I was working on. PTAL so we can merge when you have time.

Reply:
np! Thanks for working on this